src/server/game/Spells/SpellEffects.cpp

```diff
@@ -4282,7 +4282,7 @@ void Spell::EffectLeapBack(SpellEffIndex /*effIndex*/)
         return;
 
     float speedxy = effectInfo->MiscValue / 10.f;
-    float speedz = damage / 10.f;
+    float speedz = damage / 5.0f;
```

Comment:
not documented change: relevant for wild charge/travel form

Reply:
Reverted back. That function isn't correct either way ,but we need more info to understand how it should work.